### PR TITLE
Add unit tests for hub_containerbuild plugin

### PR DESCRIPTION
**Pull request by chmeliik**
Should the new test file have a copyright header like the exististing `test_kcb.py` file?

### CODE REVIEW

**Comment by MartinBasti** [comment]
> Should the new test file have a copyright header like the exististing test_kcb.py file?
Yes please.
However LICENSE file and code files contain LGPL license, test_kcb contains BSD license. I'd go with LGPL License in header.

File: tests/test_hub_containerbuild.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+This library is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public
+License along with this library; if not, write to the Free Software
+Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+"""
```
Comment (chmeliik) [inline tests/test_hub_containerbuild.py]: @MartinBasti like this?

Comment (MartinBasti) [reply tests/test_hub_containerbuild.py]: +1

**Comment by MartinBasti** [comment]
No release notes needed, test change only

**Pull request merged**